.devcontainer/devcontainer.json

```diff
@@ -5,12 +5,19 @@
     // Or use a Dockerfile or Docker Compose file. More info: https://containers.dev/guide/dockerfile
     "image": "mcr.microsoft.com/devcontainers/base:bookworm",
     "features": {
-        "ghcr.io/devcontainers/features/docker-inside-docker:1": {},
-        "ghcr.io/devcontainers/features/git:1": {},
+        "ghcr.io/devcontainers/features/docker-in-docker:2": {},
         "ghcr.io/devcontainers/features/kubectl-helm-minikube:1": {},
         "ghcr.io/devcontainers/features/python:1": {"version": "3.11"}
```

Comment:
Add option , "installTools": false

Reply:
This option should be inside `docker-in-docker` feature?

Reply:
Added `installTool` option as false